tex/gregoriotex.lua

```diff
@@ -214,7 +214,8 @@ local function init(arg, enable_height_computation)
   if enable_height_computation then
     new_line_heights = {}
 
-    local mcb_version = luatexbase.get_module_version('luatexbase-mcb')
+    local mcb_version = luatexbase.get_module_version and
+        luatexbase.get_module_version('luatexbase-mcb') or 9999
```

Comment:
Well, if this is the "standard" solution, I guess it's okay.  Personally I find it to be sloppy to rely on operator precedence for the logical operators because their precedence order isn't as well standardized as the precedence for mathematical operators, but I can live with it.